logical/src/main/java/org/apache/drill/common/config/LogicalPlanPersistence.java

```diff
@@ -52,6 +53,7 @@ public LogicalPlanPersistence(DrillConfig conf, ScanResult scanResult) {
     mapper.configure(Feature.ALLOW_UNQUOTED_FIELD_NAMES, true);
     mapper.configure(JsonGenerator.Feature.QUOTE_FIELD_NAMES, true);
     mapper.configure(Feature.ALLOW_COMMENTS, true);
+    mapper.setFilterProvider(new SimpleFilterProvider().setFailOnUnknownId(false));
```

Comment:
Will filtering passwords work when profiles are sent between nodes (i.e. when we have several major fragments)?

Reply:
submitted physical plan directly to node, it was successfully deserialized